Makefile

```diff
@@ -5,6 +5,14 @@ install:
 	npm --version; node --version;
 	npm ci;
 
+code-coverage-before-build:
+	curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter;
+	chmod +x ./cc-test-reporter;
+	./cc-test-reporter before-build;
+
+code-coverage-after-build:
+	./cc-test-reporter after-build -t lcov --debug;
```

Comment:
```suggestion
	./cc-test-reporter after-build -t lcov;
```
I'd suggest removing the debug flag, since the output is near-unreadable.
```
+ make code-coverage-after-build

./cc-test-reporter after-build -t lcov --debug;

time="2018-12-14T11:24:36Z" level=debug msg="about to run format-coverage" 

time="2018-12-14T11:24:36Z" level=debug msg="using formatter lcov" 

time="2018-12-14T11:24:36Z" level=debug msg="checking search path  for lcov formatter" 

time="2018-12-14T11:24:36Z" level=debug msg="checking search path coverage/lcov.info for lcov formatter" 

time="2018-12-14T11:24:36Z" level=debug msg="couldn't load committed at from ENV, trying git..." 

time="2018-12-14T11:24:36Z" level=info msg="trimming with prefix /var/lib/jenkins/workspace/sammead_code-climate-readme-VLRX2RYNGMADOZITW2MR33WISMPRF67FAZ5RLTPW6FO3RXIEX2SA@2/" 

time="2018-12-14T11:24:36Z" level=debug msg="getting git blob_id for source file packages/components/example-button/src/index.jsx" 

time="2018-12-14T11:24:36Z" level=info msg="trimming with prefix /var/lib/jenkins/workspace/sammead_code-climate-readme-VLRX2RYNGMADOZITW2MR33WISMPRF67FAZ5RLTPW6FO3RXIEX2SA@2/" 

time="2018-12-14T11:24:36Z" level=debug msg="getting git blob_id for source file packages/components/example-button/src/styles/styledExampleButton.js" 

time="2018-12-14T11:24:36Z" level=info msg="trimming with prefix /var/lib/jenkins/workspace/sammead_code-climate-readme-VLRX2RYNGMADOZITW2MR33WISMPRF67FAZ5RLTPW6FO3RXIEX2SA@2/" 

time="2018-12-14T11:24:36Z" level=debug msg="getting git blob_id for source file packages/components/psammead-brand/src/index.jsx" 

time="2018-12-14T11:24:36Z" level=info msg="trimming with prefix /var/lib/jenkins/workspace/sammead_code-climate-readme-VLRX2RYNGMADOZITW2MR33WISMPRF67FAZ5RLTPW6FO3RXIEX2SA@2/" 

time="2018-12-14T11:24:36Z" level=debug msg="getting git blob_id for source file packages/components/psammead-caption/src/index.jsx" 

time="2018-12-14T11:24:36Z" level=info msg="trimming with prefix /var/lib/jenkins/workspace/sammead_code-climate-readme-VLRX2RYNGMADOZITW2MR33WISMPRF67FAZ5RLTPW6FO3RXIEX2SA@2/" 

time="2018-12-14T11:24:36Z" level=debug msg="getting git blob_id for source file packages/components/psammead-copyright/src/index.jsx" 

time="2018-12-14T11:24:36Z" level=info msg="trimming with prefix /var/lib/jenkins/workspace/sammead_code-climate-readme-VLRX2RYNGMADOZITW2MR33WISMPRF67FAZ5RLTPW6FO3RXIEX2SA@2/" 

time="2018-12-14T11:24:36Z" level=debug msg="getting git blob_id for source file packages/components/psammead-headings/src/index.jsx" 

time="2018-12-14T11:24:36Z" level=info msg="trimming with prefix /var/lib/jenkins/workspace/sammead_code-climate-readme-VLRX2RYNGMADOZITW2MR33WISMPRF67FAZ5RLTPW6FO3RXIEX2SA@2/" 

time="2018-12-14T11:24:36Z" level=debug msg="getting git blob_id for source file packages/components/psammead-inline-link/src/index.jsx" 

time="2018-12-14T11:24:36Z" level=info msg="trimming with prefix /var/lib/jenkins/workspace/sammead_code-climate-readme-VLRX2RYNGMADOZITW2MR33WISMPRF67FAZ5RLTPW6FO3RXIEX2SA@2/" 

time="2018-12-14T11:24:36Z" level=debug msg="getting git blob_id for source file packages/components/psammead-paragraph/src/index.jsx" 

time="2018-12-14T11:24:36Z" level=info msg="trimming with prefix /var/lib/jenkins/workspace/sammead_code-climate-readme-VLRX2RYNGMADOZITW2MR33WISMPRF67FAZ5RLTPW6FO3RXIEX2SA@2/" 

time="2018-12-14T11:24:36Z" level=debug msg="getting git blob_id for source file packages/components/psammead-sitewide-links/src/index.jsx" 

time="2018-12-14T11:24:36Z" level=info msg="trimming with prefix /var/lib/jenkins/workspace/sammead_code-climate-readme-VLRX2RYNGMADOZITW2MR33WISMPRF67FAZ5RLTPW6FO3RXIEX2SA@2/" 

time="2018-12-14T11:24:36Z" level=debug msg="getting git blob_id for source file packages/components/psammead-sitewide-links/src/Link/index.jsx" 

time="2018-12-14T11:24:36Z" level=info msg="trimming with prefix /var/lib/jenkins/workspace/sammead_code-climate-readme-VLRX2RYNGMADOZITW2MR33WISMPRF67FAZ5RLTPW6FO3RXIEX2SA@2/" 

time="2018-12-14T11:24:36Z" level=debug msg="getting git blob_id for source file packages/components/psammead-sitewide-links/src/List/index.jsx" 

time="2018-12-14T11:24:36Z" level=info msg="trimming with prefix /var/lib/jenkins/workspace/sammead_code-climate-readme-VLRX2RYNGMADOZITW2MR33WISMPRF67FAZ5RLTPW6FO3RXIEX2SA@2/" 

time="2018-12-14T11:24:36Z" level=debug msg="getting git blob_id for source file packages/components/psammead-visually-hidden-text/src/index.jsx" 

time="2018-12-14T11:24:36Z" level=info msg="trimming with prefix /var/lib/jenkins/workspace/sammead_code-climate-readme-VLRX2RYNGMADOZITW2MR33WISMPRF67FAZ5RLTPW6FO3RXIEX2SA@2/" 

time="2018-12-14T11:24:36Z" level=debug msg="getting git blob_id for source file packages/utilities/gel-constants/src/breakpoints.js" 

time="2018-12-14T11:24:36Z" level=info msg="trimming with prefix /var/lib/jenkins/workspace/sammead_code-climate-readme-VLRX2RYNGMADOZITW2MR33WISMPRF67FAZ5RLTPW6FO3RXIEX2SA@2/" 

time="2018-12-14T11:24:36Z" level=debug msg="getting git blob_id for source file packages/utilities/gel-constants/src/spacings.js" 

time="2018-12-14T11:24:36Z" level=info msg="trimming with prefix /var/lib/jenkins/workspace/sammead_code-climate-readme-VLRX2RYNGMADOZITW2MR33WISMPRF67FAZ5RLTPW6FO3RXIEX2SA@2/" 

time="2018-12-14T11:24:36Z" level=debug msg="getting git blob_id for source file packages/utilities/gel-foundations-styled-components/src/typography.js" 

time="2018-12-14T11:24:36Z" level=info msg="trimming with prefix /var/lib/jenkins/workspace/sammead_code-climate-readme-VLRX2RYNGMADOZITW2MR33WISMPRF67FAZ5RLTPW6FO3RXIEX2SA@2/" 

time="2018-12-14T11:24:36Z" level=debug msg="getting git blob_id for source file packages/utilities/psammead-assets/src/amp-boilerplate.js" 

time="2018-12-14T11:24:36Z" level=info msg="trimming with prefix /var/lib/jenkins/workspace/sammead_code-climate-readme-VLRX2RYNGMADOZITW2MR33WISMPRF67FAZ5RLTPW6FO3RXIEX2SA@2/" 

time="2018-12-14T11:24:36Z" level=debug msg="getting git blob_id for source file packages/utilities/psammead-assets/src/svgs.js" 

time="2018-12-14T11:24:36Z" level=info msg="trimming with prefix /var/lib/jenkins/workspace/sammead_code-climate-readme-VLRX2RYNGMADOZITW2MR33WISMPRF67FAZ5RLTPW6FO3RXIEX2SA@2/" 

time="2018-12-14T11:24:36Z" level=debug msg="getting git blob_id for source file packages/utilities/psammead-styles/src/colours.js" 

time="2018-12-14T11:24:36Z" level=info msg="trimming with prefix /var/lib/jenkins/workspace/sammead_code-climate-readme-VLRX2RYNGMADOZITW2MR33WISMPRF67FAZ5RLTPW6FO3RXIEX2SA@2/" 

time="2018-12-14T11:24:36Z" level=debug msg="getting git blob_id for source file packages/utilities/psammead-styles/src/fonts.js" 

time="2018-12-14T11:24:36Z" level=info msg="trimming with prefix /var/lib/jenkins/workspace/sammead_code-climate-readme-VLRX2RYNGMADOZITW2MR33WISMPRF67FAZ5RLTPW6FO3RXIEX2SA@2/" 

time="2018-12-14T11:24:36Z" level=debug msg="getting git blob_id for source file packages/utilities/psammead-test-helpers/src/index.js" 

time="2018-12-14T11:24:36Z" level=info msg="trimming with prefix /var/lib/jenkins/workspace/sammead_code-climate-readme-VLRX2RYNGMADOZITW2MR33WISMPRF67FAZ5RLTPW6FO3RXIEX2SA@2/" 

time="2018-12-14T11:24:36Z" level=debug msg="getting git blob_id for source file scripts/publish/index.js" 

time="2018-12-14T11:24:36Z" level=info msg="trimming with prefix /var/lib/jenkins/workspace/sammead_code-climate-readme-VLRX2RYNGMADOZITW2MR33WISMPRF67FAZ5RLTPW6FO3RXIEX2SA@2/" 

time="2018-12-14T11:24:36Z" level=debug msg="getting git blob_id for source file scripts/publish/src/getPackages.js" 

time="2018-12-14T11:24:36Z" level=info msg="trimming with prefix /var/lib/jenkins/workspace/sammead_code-climate-readme-VLRX2RYNGMADOZITW2MR33WISMPRF67FAZ5RLTPW6FO3RXIEX2SA@2/" 

time="2018-12-14T11:24:36Z" level=debug msg="getting git blob_id for source file scripts/publish/src/getRegistryVersion.js" 

time="2018-12-14T11:24:36Z" level=info msg="trimming with prefix /var/lib/jenkins/workspace/sammead_code-climate-readme-VLRX2RYNGMADOZITW2MR33WISMPRF67FAZ5RLTPW6FO3RXIEX2SA@2/" 

time="2018-12-14T11:24:36Z" level=debug msg="getting git blob_id for source file scripts/publish/src/publish.js" 

time="2018-12-14T11:24:36Z" level=info msg="trimming with prefix /var/lib/jenkins/workspace/sammead_code-climate-readme-VLRX2RYNGMADOZITW2MR33WISMPRF67FAZ5RLTPW6FO3RXIEX2SA@2/" 

time="2018-12-14T11:24:36Z" level=debug msg="getting git blob_id for source file scripts/publish/src/publishConfig.js" 

time="2018-12-14T11:24:36Z" level=info msg="trimming with prefix /var/lib/jenkins/workspace/sammead_code-climate-readme-VLRX2RYNGMADOZITW2MR33WISMPRF67FAZ5RLTPW6FO3RXIEX2SA@2/" 

time="2018-12-14T11:24:36Z" level=debug msg="getting git blob_id for source file scripts/publish/src/report.js" 

time="2018-12-14T11:24:36Z" level=info msg="trimming with prefix /var/lib/jenkins/workspace/sammead_code-climate-readme-VLRX2RYNGMADOZITW2MR33WISMPRF67FAZ5RLTPW6FO3RXIEX2SA@2/" 

time="2018-12-14T11:24:36Z" level=debug msg="getting git blob_id for source file scripts/publish/src/shouldPublish.js" 

time="2018-12-14T11:24:36Z" level=info msg="trimming with prefix /var/lib/jenkins/workspace/sammead_code-climate-readme-VLRX2RYNGMADOZITW2MR33WISMPRF67FAZ5RLTPW6FO3RXIEX2SA@2/" 

time="2018-12-14T11:24:36Z" level=debug msg="getting git blob_id for source file scripts/publish/src/slackNotification.js" 

time="2018-12-14T11:24:36Z" level=debug msg="codeclimate.json content: {\n  \"ci_service\": {\n    \"branch\": \"code-climate-readme\",\n    \"build_identifier\": \"46\",\n    \"build_url\": \"https://ci.news.tools.bbc.co.uk/job/psammead/job/code-climate-readme/46/\",\n    \"commit_sha\": \"412a34c69907ba209845dc248429e0d09525d1a9\",\n    \"committed_at\": 1544786595,\n    \"name\": \"https://ci.news.tools.bbc.co.uk/\"\n  },\n  \"environment\": {\n    \"gem_version\": \"\",\n    \"package_version\": \"\",\n    \"pwd\": \"/var/lib/jenkins/workspace/sammead_code-climate-readme-VLRX2RYNGMADOZITW2MR33WISMPRF67FAZ5RLTPW6FO3RXIEX2SA@2\",\n    \"prefix\": \"/var/lib/jenkins/workspace/sammead_code-climate-readme-VLRX2RYNGMADOZITW2MR33WISMPRF67FAZ5RLTPW6FO3RXIEX2SA@2\",\n    \"rails_root\": \"\",\n    \"reporter_version\": \"0.6.3\",\n    \"simplecov_root\": \"\"\n  },\n  \"git\": {\n    \"branch\": \"code-climate-readme\",\n    \"head\": \"412a34c69907ba209845dc248429e0d09525d1a9\",\n    \"committed_at\": 1544786595\n  },\n  \"covered_percent\": 55.70776255707762,\n  \"covered_strength\": 0,\n  \"line_counts\": {\n    \"missed\": 97,\n    \"covered\": 122,\n    \"total\": 219\n  },\n  \"source_files\": [\n    {\n      \"blob_id\": \"56464e5d35bd61e1f1a86c39070c4f35228e04a7\",\n      \"coverage\": \"[null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,1,null,null,null,null,null,null,null,null,1,null,null,null,null,null,null,1,null,null,null,null,null,1,1,null,null,null,null,null,null,null,null,1,null,null,null,null,1]\",\n      \"covered_percent\": 100,\n      \"covered_strength\": 1,\n      \"line_counts\": {\n        \"missed\": 0,\n        \"covered\": 7,\n        \"total\": 7\n      },\n      \"name\": \"packages/components/psammead-sitewide-links/src/index.jsx\"\n    },\n    {\n      \"blob_id\": \"b3d9d1eca54883e85de72a1fe218f253286a2025\",\n      \"coverage\": \"[null,null,null,null,null,0,0,0,0,0,null,null,null,null,null,null,0,0,0,0]\",\n      \"covered_percent\": 0,\n      \"covered_strength\": 0,\n      \"line_counts\": {\n        \"missed\": 9,\n        \"covered\": 0,\n        \"total\": 9\n      },\n      \"name\": \"packages/utilities/gel-constants/src/spacings.js\"\n    },\n    {\n      \"blob_id\": \"4fb06c4e5eac0bef6ead00aa596f772f6fe3b539\",\n      \"coverage\": \"[0,0,0,0,0,null,0,null,0,0,null,0,0,null,null,null,0,null,0]\",\n      \"covered_percent\": 0,\n      \"covered_strength\": 0,\n      \"line_counts\": {\n        \"missed\": 12,\n        \"covered\": 0,\n        \"total\": 12\n      },\n      \"name\": \"scripts/publish/index.js\"\n    },\n    {\n      \"blob_id\": \"0017151328a4fdaab626a5a41403b4383c44955f\",\n      \"coverage\": \"[null,null,null,null,null,null,null,null,null,null,null,null,1,null,null,null,null,null,null,null,1,1,null,4]\",\n      \"covered_percent\": 100,\n      \"covered_strength\": 1.75,\n      \"line_counts\": {\n        \"missed\": 0,\n        \"covered\": 4,\n        \"total\": 4\n      },\n      \"name\": \"packages/components/psammead-headings/src/index.jsx\"\n    },\n    {\n      \"blob_id\": \"b06b4de973694b610a3211478df20401d4c983ba\",\n      \"coverage\": \"[null,null,null,null,null,null,null,null,1]\",\n      \"covered_percent\": 100,\n      \"covered_strength\": 1,\n      \"line_counts\": {\n        \"missed\": 0,\n        \"covered\": 1,\n        \"total\": 1\n      },\n      \"name\": \"packages/components/psammead-inline-link/src/index.jsx\"\n    },\n    {\n      \"blob_id\": \"e867f8294ec71c4b4fd5522461368ba56a56194f\",\n      \"coverage\": \"[0,0,0,null,0,0,0,0,null,null,null,0,0,0,0,null,0,null,null,0,0,0,0,null,null,null,0,0,0,0]\",\n      \"covered_percent\": 0,\n      \"covered_strength\": 0,\n      \"line_counts\": {\n        \"missed\": 20,\n        \"covered\": 0,\n        \"total\": 20\n      },\n      \"name\": \"packages/utilities/psammead-test-helpers/src/index.js\"\n    },\n    {\n      \"blob_id\": \"82d14d3b859f669e9198219711d9a9f798caa262\",\n      \"coverage\": \"[3,null,null,3,3,3]\",\n      \"covered_percent\": 100,\n      \"covered_strength\": 3,\n      \"line_counts\": {\n        \"missed\": 0,\n        \"covered\": 4,\n        \"total\": 4\n      },\n      \"name\": \"scripts/publish/src/getRegistryVersion.js\"\n    },\n    {\n      \"blob_id\": \"00746eb04b1513e85a94c4dc19bdb160d73ba18f\",\n      \"coverage\": \"[null,null,null,null,null,null,1]\",\n      \"covered_percent\": 100,\n      \"covered_strength\": 1,\n      \"line_counts\": {\n        \"missed\": 0,\n        \"covered\": 1,\n        \"total\": 1\n      },\n      \"name\": \"packages/components/psammead-paragraph/src/index.jsx\"\n    },\n    {\n      \"blob_id\": \"d2fee701a6db80d385aa65631a780b26aae61cfa\",\n      \"coverage\": \"[null,null,null,null,0,0]\",\n      \"covered_percent\": 0,\n      \"covered_strength\": 0,\n      \"line_counts\": {\n        \"missed\": 2,\n        \"covered\": 0,\n        \"total\": 2\n      },\n      \"name\": \"packages/utilities/psammead-assets/src/amp-boilerplate.js\"\n    },\n    {\n      \"blob_id\": \"8cf13bdfcb29471c7fa47be092245fcc82359e16\",\n      \"coverage\": \"[null,null,1]\",\n      \"covered_percent\": 100,\n      \"covered_strength\": 1,\n      \"line_counts\": {\n        \"missed\": 0,\n        \"covered\": 1,\n        \"total\": 1\n      },\n      \"name\": \"packages/components/psammead-visually-hidden-text/src/index.jsx\"\n    },\n    {\n      \"blob_id\": \"b018ec89f2d04964903de900ef343aee499b4d46\",\n      \"coverage\": \"[3,null,null,3,3,null,5]\",\n      \"covered_percent\": 100,\n      \"covered_strength\": 3.5,\n      \"line_counts\": {\n        \"missed\": 0,\n        \"covered\": 4,\n        \"total\": 4\n      },\n      \"name\": \"scripts/publish/src/getPackages.js\"\n    },\n    {\n      \"blob_id\": \"276927d4065eeb296a0c52a988b2736c5ad1a57b\",\n      \"coverage\": \"[null,3,3,3,3,3,null,3,3,null,3,null,3,null,3,null,3,null,null,null,null,null,null,3,2,2,2,2,null,1,1,1]\",\n      \"covered_percent\": 100,\n      \"covered_strength\": 2.473684210526316,\n      \"line_counts\": {\n        \"missed\": 0,\n        \"covered\": 19,\n        \"total\": 19\n      },\n      \"name\": \"scripts/publish/src/publish.js\"\n    },\n    {\n      \"blob_id\": \"0de81dd344d00da26065ff81fc0baf397863afbc\",\n      \"coverage\": \"[null,null,null,null,null,null,null,null,3,3]\",\n      \"covered_percent\": 100,\n      \"covered_strength\": 3,\n      \"line_counts\": {\n        \"missed\": 0,\n        \"covered\": 2,\n        \"total\": 2\n      },\n      \"name\": \"packages/components/example-button/src/styles/styledExampleButton.js\"\n    },\n    {\n      \"blob_id\": \"fdc1c2f3847be6b51bdf627e905c7722eb1963ec\",\n      \"coverage\": \"[null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,2,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,2,null,null,null,null,null,null,2,2,null,2,null,null,null,null,null,null,2,null,null,null,null,2]\",\n      \"covered_percent\": 100,\n      \"covered_strength\": 2,\n      \"line_counts\": {\n        \"missed\": 0,\n        \"covered\": 7,\n        \"total\": 7\n      },\n      \"name\": \"packages/components/psammead-sitewide-links/src/List/index.jsx\"\n    },\n    {\n      \"blob_id\": \"ae2e5389765a70e8271545dae5be07806e58ac07\",\n      \"coverage\": \"[null,null,null,null,null,null,3,null,null,null,null,10,null,null,3,null,null,null,null,null,null,null,3,5,null,null,null,null,3,null,null,null,3]\",\n      \"covered_percent\": 100,\n      \"covered_strength\": 4.285714285714286,\n      \"line_counts\": {\n        \"missed\": 0,\n        \"covered\": 7,\n        \"total\": 7\n      },\n      \"name\": \"packages/components/psammead-sitewide-links/src/Link/index.jsx\"\n    },\n    {\n      \"blob_id\": \"129ede87bb1b2146e78a9cb79efa5108085bbca5\",\n      \"coverage\": \"[null,0,0,0,0,0,0,0,0,0,0,0,null,null,0]\",\n      \"covered_percent\": 0,\n      \"covered_strength\": 0,\n      \"line_counts\": {\n        \"missed\": 12,\n        \"covered\": 0,\n        \"total\": 12\n      },\n      \"name\": \"packages/utilities/psammead-styles/src/colours.js\"\n    },\n    {\n      \"blob_id\": \"77241af32e6a26fa6fe1fd6ce60b546e57ef6a0c\",\n      \"coverage\": \"[4,7,7,null,null,7,3,2,null,null,3,2,null,null,null,7]\",\n      \"covered_percent\": 100,\n      \"covered_strength\": 4.666666666666667,\n      \"line_counts\": {\n        \"missed\": 0,\n        \"covered\": 9,\n        \"total\": 9\n      },\n      \"name\": \"scripts/publish/src/publishConfig.js\"\n    },\n    {\n      \"blob_id\": \"d8ca4856baaf58baabd7677fa5c30e74e6f38b0c\",\n      \"coverage\": \"[1,1,null,1,4]\",\n      \"covered_percent\": 100,\n      \"covered_strength\": 1.75,\n      \"line_counts\": {\n        \"missed\": 0,\n        \"covered\": 4,\n        \"total\": 4\n      },\n      \"name\": \"scripts/publish/src/shouldPublish.js\"\n    },\n    {\n      \"blob_id\": \"5f311fa2b4e3be4865d8718658865597d4964add\",\n      \"coverage\": \"[null,null,0]\",\n      \"covered_percent\": 0,\n      \"covered_strength\": 0,\n      \"line_counts\": {\n        \"missed\": 1,\n        \"covered\": 0,\n        \"total\": 1\n      },\n      \"name\": \"packages/utilities/psammead-assets/src/svgs.js\"\n    },\n    {\n      \"blob_id\": \"202436c5321f5255f915e64efd74c577d3c50ad3\",\n      \"coverage\": \"[null,0,0,0,0,0,0,0,0,0]\",\n      \"covered_percent\": 0,\n      \"covered_strength\": 0,\n      \"line_counts\": {\n        \"missed\": 9,\n        \"covered\": 0,\n        \"total\": 9\n      },\n      \"name\": \"packages/utilities/psammead-styles/src/fonts.js\"\n    },\n    {\n      \"blob_id\": \"0ad607c26bf16fbd2ffebfd3108cd08393524976\",\n      \"coverage\": \"[null,null,null,null,null,null,null,1]\",\n      \"covered_percent\": 100,\n      \"covered_strength\": 1,\n      \"line_counts\": {\n        \"missed\": 0,\n        \"covered\": 1,\n        \"total\": 1\n      },\n      \"name\": \"packages/components/psammead-caption/src/index.jsx\"\n    },\n    {\n      \"blob_id\": \"c0efc628d3e2b47a5e94f510846cc08850769d31\",\n      \"coverage\": \"[null,null,null,null,null,null,1]\",\n      \"covered_percent\": 100,\n      \"covered_strength\": 1,\n      \"line_counts\": {\n        \"missed\": 0,\n        \"covered\": 1,\n        \"total\": 1\n      },\n      \"name\": \"packages/components/psammead-copyright/src/index.jsx\"\n    },\n    {\n      \"blob_id\": \"89fa93f127ad63b29a9ded2e7ab457bb5663aebd\",\n      \"coverage\": \"[0,0,null,0,null,null,null,null,null,null,null,null,null,null,null,null,null,null,0,null,null,null,null,null,null,null,null,null,null,null,null,null,null,0,null,null,null,null,null,null,null,null,null,null,null,null,null,null,0,null,null,null,null,null,null,null,null,null,null,null,null,null,null,0,null,null,null,null,null,null,null,null,null,null,null,null,null,null,0,null,null,null,null,null,null,null,null,null,null,null,null,null,null,0,null,null,null,null,null,null,null,null,null,null,null,null,null,null,0,null,null,null,null,null,null,null,null,null,null,null,null,null,null,0,null,null,null,null,null,null,null,null,null,0,null,null,null,null,null,null,null,null,null,0,null,null,null,null,null,null,null,null,null,0,null,null,null,null,null,null,null,null,null,0,null,null,null,null,null,null,null,null,0,null,null,null,null,null,null,null,null,null,null,null,null,0]\",\n      \"covered_percent\": 0,\n      \"covered_strength\": 0,\n      \"line_counts\": {\n        \"missed\": 17,\n        \"covered\": 0,\n        \"total\": 17\n      },\n      \"name\": \"packages/utilities/gel-foundations-styled-components/src/typography.js\"\n    },\n    {\n      \"blob_id\": \"e43e1eb98a82654ca7cc666bfa433f47bc3244b0\",\n      \"coverage\": \"[null,1,null,1,4,4,4,null,4,null,null,null,4,null,null,null,4,null,null,null,4,2,2,3,null,null,null,4,2,2,null,null,2]\",\n      \"covered_percent\": 100,\n      \"covered_strength\": 2.9375,\n      \"line_counts\": {\n        \"missed\": 0,\n        \"covered\": 16,\n        \"total\": 16\n      },\n      \"name\": \"scripts/publish/src/report.js\"\n    },\n    {\n      \"blob_id\": \"04442f5dfdabdbab78a6252bb4b452a717843819\",\n      \"coverage\": \"[null,5,5,null,5,4,null,4,3,null,null,null,3,null,3,null,3,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,3,1,1,1,null,null,null,1]\",\n      \"covered_percent\": 100,\n      \"covered_strength\": 3,\n      \"line_counts\": {\n        \"missed\": 0,\n        \"covered\": 14,\n        \"total\": 14\n      },\n      \"name\": \"scripts/publish/src/slackNotification.js\"\n    },\n    {\n      \"blob_id\": \"474577360ef634d9a3e73b4c22a62b8601072b09\",\n      \"coverage\": \"[null,null,null,null,1,null,null,null,null,null,3,null,null,null,null,null,null,null,null,null,1,null,null,null,null,null,null,1]\",\n      \"covered_percent\": 100,\n      \"covered_strength\": 1.5,\n      \"line_counts\": {\n        \"missed\": 0,\n        \"covered\": 4,\n        \"total\": 4\n      },\n      \"name\": \"packages/components/example-button/src/index.jsx\"\n    },\n    {\n      \"blob_id\": \"93feba0c710cf8cc4ae5e83ab54bb4cd436e3930\",\n      \"coverage\": \"[null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,1,null,null,null,null,null,null,null,null,1,1,1,null,1,1,1,1,null,1,null,1,null,null,null,null,null,null,null,null,null,1,null,null,null,null,1,null,null,null,null,null,null,null,null,null,1,null,null,null,null,null,null,1,1,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,1]\",\n      \"covered_percent\": 100,\n      \"covered_strength\": 1,\n      \"line_counts\": {\n        \"missed\": 0,\n        \"covered\": 16,\n        \"total\": 16\n      },\n      \"name\": \"packages/components/psammead-brand/src/index.jsx\"\n    },\n    {\n      \"blob_id\": \"478ea37bdd6bfb590009d917979651277c1dd596\",\n      \"coverage\": \"[null,null,null,null,null,0,0,0,null,null,null,null,null,null,0,0,null,0,0,null,0,0,null,0,0,null,0,0,null,0,null,0]\",\n      \"covered_percent\": 0,\n      \"covered_strength\": 0,\n      \"line_counts\": {\n        \"missed\": 15,\n        \"covered\": 0,\n        \"total\": 15\n      },\n      \"name\": \"packages/utilities/gel-constants/src/breakpoints.js\"\n    }\n  ],\n  \"repo_token\": \"****\"\n}" 

time="2018-12-14T11:24:36Z" level=debug msg="about to run upload-coverage" 

time="2018-12-14T11:24:36Z" level=debug msg="posting request to https://api.codeclimate.com/v1/test_reports" 

time="2018-12-14T11:24:37Z" level=debug msg="{\"data\":{\"type\":\"test_reports\",\"attributes\":{\"ci_branch\":\"code-climate-readme\",\"ci_build_identifier\":\"46\",\"ci_build_url\":\"https://ci.news.tools.bbc.co.uk/job/psammead/job/code-climate-readme/46/\",\"ci_commit_sha\":\"412a34c69907ba209845dc248429e0d09525d1a9\",\"ci_committed_at\":1544786595,\"ci_service_name\":\"https://ci.news.tools.bbc.co.uk/\",\"git_branch\":\"code-climate-readme\",\"commit_sha\":\"412a34c69907ba209845dc248429e0d09525d1a9\",\"committed_at\":1544786595,\"run_at\":1544786676,\"covered_percent\":55.70776255707762,\"covered_strength\":0,\"environment\":{\"gem_version\":\"\",\"package_version\":\"\",\"pwd\":\"/var/lib/jenkins/workspace/sammead_code-climate-readme-VLRX2RYNGMADOZITW2MR33WISMPRF67FAZ5RLTPW6FO3RXIEX2SA@2\",\"prefix\":\"/var/lib/jenkins/workspace/sammead_code-climate-readme-VLRX2RYNGMADOZITW2MR33WISMPRF67FAZ5RLTPW6FO3RXIEX2SA@2\",\"rails_root\":\"\",\"reporter_version\":\"0.6.3\",\"simplecov_root\":\"\"},\"line_counts\":{\"missed\":97,\"covered\":122,\"total\":219}}},\"meta\":null}\n" 

time="2018-12-14T11:24:37Z" level=debug msg="posting request to https://api.codeclimate.com/v1/repos/5c000858f9efa63c0e003c68/test_reports/5c1392f5a6da79133a001b78/test_file_reports/batch" 

time="2018-12-14T11:24:37Z" level=debug msg="{\"data\":[{\"type\":\"test_file_reports\",\"blob_id\":\"d2fee701a6db80d385aa65631a780b26aae61cfa\",\"coverage\":\"[null,null,null,null,0,0]\",\"covered_percent\":0,\"covered_strength\":0,\"line_counts\":{\"missed\":2,\"covered\":0,\"total\":2},\"path\":\"packages/utilities/psammead-assets/src/amp-boilerplate.js\"},{\"type\":\"test_file_reports\",\"blob_id\":\"89fa93f127ad63b29a9ded2e7ab457bb5663aebd\",\"coverage\":\"[0,0,null,0,null,null,null,null,null,null,null,null,null,null,null,null,null,null,0,null,null,null,null,null,null,null,null,null,null,null,null,null,null,0,null,null,null,null,null,null,null,null,null,null,null,null,null,null,0,null,null,null,null,null,null,null,null,null,null,null,null,null,null,0,null,null,null,null,null,null,null,null,null,null,null,null,null,null,0,null,null,null,null,null,null,null,null,null,null,null,null,null,null,0,null,null,null,null,null,null,null,null,null,null,null,null,null,null,0,null,null,null,null,null,null,null,null,null,null,null,null,null,null,0,null,null,null,null,null,null,null,null,null,0,null,null,null,null,null,null,null,null,null,0,null,null,null,null,null,null,null,null,null,0,null,null,null,null,null,null,null,null,null,0,null,null,null,null,null,null,null,null,0,null,null,null,null,null,null,null,null,null,null,null,null,0]\",\"covered_percent\":0,\"covered_strength\":0,\"line_counts\":{\"missed\":17,\"covered\":0,\"total\":17},\"path\":\"packages/utilities/gel-foundations-styled-components/src/typography.js\"},{\"type\":\"test_file_reports\",\"blob_id\":\"4fb06c4e5eac0bef6ead00aa596f772f6fe3b539\",\"coverage\":\"[0,0,0,0,0,null,0,null,0,0,null,0,0,null,null,null,0,null,0]\",\"covered_percent\":0,\"covered_strength\":0,\"line_counts\":{\"missed\":12,\"covered\":0,\"total\":12},\"path\":\"scripts/publish/index.js\"},{\"type\":\"test_file_reports\",\"blob_id\":\"82d14d3b859f669e9198219711d9a9f798caa262\",\"coverage\":\"[3,null,null,3,3,3]\",\"covered_percent\":100,\"covered_strength\":3,\"line_counts\":{\"missed\":0,\"covered\":4,\"total\":4},\"path\":\"scripts/publish/src/getRegistryVersion.js\"},{\"type\":\"test_file_reports\",\"blob_id\":\"77241af32e6a26fa6fe1fd6ce60b546e57ef6a0c\",\"coverage\":\"[4,7,7,null,null,7,3,2,null,null,3,2,null,null,null,7]\",\"covered_percent\":100,\"covered_strength\":4.666666666666667,\"line_counts\":{\"missed\":0,\"covered\":9,\"total\":9},\"path\":\"scripts/publish/src/publishConfig.js\"},{\"type\":\"test_file_reports\",\"blob_id\":\"202436c5321f5255f915e64efd74c577d3c50ad3\",\"coverage\":\"[null,0,0,0,0,0,0,0,0,0]\",\"covered_percent\":0,\"covered_strength\":0,\"line_counts\":{\"missed\":9,\"covered\":0,\"total\":9},\"path\":\"packages/utilities/psammead-styles/src/fonts.js\"},{\"type\":\"test_file_reports\",\"blob_id\":\"474577360ef634d9a3e73b4c22a62b8601072b09\",\"coverage\":\"[null,null,null,null,1,null,null,null,null,null,3,null,null,null,null,null,null,null,null,null,1,null,null,null,null,null,null,1]\",\"covered_percent\":100,\"covered_strength\":1.5,\"line_counts\":{\"missed\":0,\"covered\":4,\"total\":4},\"path\":\"packages/components/example-button/src/index.jsx\"},{\"type\":\"test_file_reports\",\"blob_id\":\"0017151328a4fdaab626a5a41403b4383c44955f\",\"coverage\":\"[null,null,null,null,null,null,null,null,null,null,null,null,1,null,null,null,null,null,null,null,1,1,null,4]\",\"covered_percent\":100,\"covered_strength\":1.75,\"line_counts\":{\"missed\":0,\"covered\":4,\"total\":4},\"path\":\"packages/components/psammead-headings/src/index.jsx\"},{\"type\":\"test_file_reports\",\"blob_id\":\"b06b4de973694b610a3211478df20401d4c983ba\",\"coverage\":\"[null,null,null,null,null,null,null,null,1]\",\"covered_percent\":100,\"covered_strength\":1,\"line_counts\":{\"missed\":0,\"covered\":1,\"total\":1},\"path\":\"packages/components/psammead-inline-link/src/index.jsx\"},{\"type\":\"test_file_reports\",\"blob_id\":\"5f311fa2b4e3be4865d8718658865597d4964add\",\"coverage\":\"[null,null,0]\",\"covered_percent\":0,\"covered_strength\":0,\"line_counts\":{\"missed\":1,\"covered\":0,\"total\":1},\"path\":\"packages/utilities/psammead-assets/src/svgs.js\"},{\"type\":\"test_file_reports\",\"blob_id\":\"276927d4065eeb296a0c52a988b2736c5ad1a57b\",\"coverage\":\"[null,3,3,3,3,3,null,3,3,null,3,null,3,null,3,null,3,null,null,null,null,null,null,3,2,2,2,2,null,1,1,1]\",\"covered_percent\":100,\"covered_strength\":2.473684210526316,\"line_counts\":{\"missed\":0,\"covered\":19,\"total\":19},\"path\":\"scripts/publish/src/publish.js\"},{\"type\":\"test_file_reports\",\"blob_id\":\"d8ca4856baaf58baabd7677fa5c30e74e6f38b0c\",\"coverage\":\"[1,1,null,1,4]\",\"covered_percent\":100,\"covered_strength\":1.75,\"line_counts\":{\"missed\":0,\"covered\":4,\"total\":4},\"path\":\"scripts/publish/src/shouldPublish.js\"},{\"type\":\"test_file_reports\",\"blob_id\":\"8cf13bdfcb29471c7fa47be092245fcc82359e16\",\"coverage\":\"[null,null,1]\",\"covered_percent\":100,\"covered_strength\":1,\"line_counts\":{\"missed\":0,\"covered\":1,\"total\":1},\"path\":\"packages/components/psammead-visually-hidden-text/src/index.jsx\"},{\"type\":\"test_file_reports\",\"blob_id\":\"0de81dd344d00da26065ff81fc0baf397863afbc\",\"coverage\":\"[null,null,null,null,null,null,null,null,3,3]\",\"covered_percent\":100,\"covered_strength\":3,\"line_counts\":{\"missed\":0,\"covered\":2,\"total\":2},\"path\":\"packages/components/example-button/src/styles/styledExampleButton.js\"},{\"type\":\"test_file_reports\",\"blob_id\":\"fdc1c2f3847be6b51bdf627e905c7722eb1963ec\",\"coverage\":\"[null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,2,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,2,null,null,null,null,null,null,2,2,null,2,null,null,null,null,null,null,2,null,null,null,null,2]\",\"covered_percent\":100,\"covered_strength\":2,\"line_counts\":{\"missed\":0,\"covered\":7,\"total\":7},\"path\":\"packages/components/psammead-sitewide-links/src/List/index.jsx\"},{\"type\":\"test_file_reports\",\"blob_id\":\"0ad607c26bf16fbd2ffebfd3108cd08393524976\",\"coverage\":\"[null,null,null,null,null,null,null,1]\",\"covered_percent\":100,\"covered_strength\":1,\"line_counts\":{\"missed\":0,\"covered\":1,\"total\":1},\"path\":\"packages/components/psammead-caption/src/index.jsx\"},{\"type\":\"test_file_reports\",\"blob_id\":\"e43e1eb98a82654ca7cc666bfa433f47bc3244b0\",\"coverage\":\"[null,1,null,1,4,4,4,null,4,null,null,null,4,null,null,null,4,null,null,null,4,2,2,3,null,null,null,4,2,2,null,null,2]\",\"covered_percent\":100,\"covered_strength\":2.9375,\"line_counts\":{\"missed\":0,\"covered\":16,\"total\":16},\"path\":\"scripts/publish/src/report.js\"},{\"type\":\"test_file_reports\",\"blob_id\":\"04442f5dfdabdbab78a6252bb4b452a717843819\",\"coverage\":\"[null,5,5,null,5,4,null,4,3,null,null,null,3,null,3,null,3,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,3,1,1,1,null,null,null,1]\",\"covered_percent\":100,\"covered_strength\":3,\"line_counts\":{\"missed\":0,\"covered\":14,\"total\":14},\"path\":\"scripts/publish/src/slackNotification.js\"},{\"type\":\"test_file_reports\",\"blob_id\":\"93feba0c710cf8cc4ae5e83ab54bb4cd436e3930\",\"coverage\":\"[null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,1,null,null,null,null,null,null,null,null,1,1,1,null,1,1,1,1,null,1,null,1,null,null,null,null,null,null,null,null,null,1,null,null,null,null,1,null,null,null,null,null,null,null,null,null,1,null,null,null,null,null,null,1,1,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,1]\",\"covered_percent\":100,\"covered_strength\":1,\"line_counts\":{\"missed\":0,\"covered\":16,\"total\":16},\"path\":\"packages/components/psammead-brand/src/index.jsx\"},{\"type\":\"test_file_reports\",\"blob_id\":\"478ea37bdd6bfb590009d917979651277c1dd596\",\"coverage\":\"[null,null,null,null,null,0,0,0,null,null,null,null,null,null,0,0,null,0,0,null,0,0,null,0,0,null,0,0,null,0,null,0]\",\"covered_percent\":0,\"covered_strength\":0,\"line_counts\":{\"missed\":15,\"covered\":0,\"total\":15},\"path\":\"packages/utilities/gel-constants/src/breakpoints.js\"},{\"type\":\"test_file_reports\",\"blob_id\":\"129ede87bb1b2146e78a9cb79efa5108085bbca5\",\"coverage\":\"[null,0,0,0,0,0,0,0,0,0,0,0,null,null,0]\",\"covered_percent\":0,\"covered_strength\":0,\"line_counts\":{\"missed\":12,\"covered\":0,\"total\":12},\"path\":\"packages/utilities/psammead-styles/src/colours.js\"},{\"type\":\"test_file_reports\",\"blob_id\":\"c0efc628d3e2b47a5e94f510846cc08850769d31\",\"coverage\":\"[null,null,null,null,null,null,1]\",\"covered_percent\":100,\"covered_strength\":1,\"line_counts\":{\"missed\":0,\"covered\":1,\"total\":1},\"path\":\"packages/components/psammead-copyright/src/index.jsx\"},{\"type\":\"test_file_reports\",\"blob_id\":\"00746eb04b1513e85a94c4dc19bdb160d73ba18f\",\"coverage\":\"[null,null,null,null,null,null,1]\",\"covered_percent\":100,\"covered_strength\":1,\"line_counts\":{\"missed\":0,\"covered\":1,\"total\":1},\"path\":\"packages/components/psammead-paragraph/src/index.jsx\"},{\"type\":\"test_file_reports\",\"blob_id\":\"ae2e5389765a70e8271545dae5be07806e58ac07\",\"coverage\":\"[null,null,null,null,null,null,3,null,null,null,null,10,null,null,3,null,null,null,null,null,null,null,3,5,null,null,null,null,3,null,null,null,3]\",\"covered_percent\":100,\"covered_strength\":4.285714285714286,\"line_counts\":{\"missed\":0,\"covered\":7,\"total\":7},\"path\":\"packages/components/psammead-sitewide-links/src/Link/index.jsx\"},{\"type\":\"test_file_reports\",\"blob_id\":\"b3d9d1eca54883e85de72a1fe218f253286a2025\",\"coverage\":\"[null,null,null,null,null,0,0,0,0,0,null,null,null,null,null,null,0,0,0,0]\",\"covered_percent\":0,\"covered_strength\":0,\"line_counts\":{\"missed\":9,\"covered\":0,\"total\":9},\"path\":\"packages/utilities/gel-constants/src/spacings.js\"},{\"type\":\"test_file_reports\",\"blob_id\":\"e867f8294ec71c4b4fd5522461368ba56a56194f\",\"coverage\":\"[0,0,0,null,0,0,0,0,null,null,null,0,0,0,0,null,0,null,null,0,0,0,0,null,null,null,0,0,0,0]\",\"covered_percent\":0,\"covered_strength\":0,\"line_counts\":{\"missed\":20,\"covered\":0,\"total\":20},\"path\":\"packages/utilities/psammead-test-helpers/src/index.js\"},{\"type\":\"test_file_reports\",\"blob_id\":\"56464e5d35bd61e1f1a86c39070c4f35228e04a7\",\"coverage\":\"[null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,1,null,null,null,null,null,null,null,null,1,null,null,null,null,null,null,1,null,null,null,null,null,1,1,null,null,null,null,null,null,null,null,1,null,null,null,null,1]\",\"covered_percent\":100,\"covered_strength\":1,\"line_counts\":{\"missed\":0,\"covered\":7,\"total\":7},\"path\":\"packages/components/psammead-sitewide-links/src/index.jsx\"},{\"type\":\"test_file_reports\",\"blob_id\":\"b018ec89f2d04964903de900ef343aee499b4d46\",\"coverage\":\"[3,null,null,3,3,null,5]\",\"covered_percent\":100,\"covered_strength\":3.5,\"line_counts\":{\"missed\":0,\"covered\":4,\"total\":4},\"path\":\"scripts/publish/src/getPackages.js\"}],\"meta\":{\"current\":1,\"total\":1}}\n" 

Test report uploaded successfully to Code Climate

```